docs/userguide.rst

```diff
@@ -64,6 +64,32 @@ Here is one example of well-working versions for REANA v0.5.0:
    virtualbox-guest-iso 6.0.6-1
    virtualbox-host-modules-arch 6.0.6-1
 
+Ubuntu
+~~~~~
+
+For example, on Ubuntu 18.04 LTS, you should install the following packages by following their respective installation guides:
```

Comment:
It would be good to mention only Ubuntu-specific setup, i.e. adding any PPAs etc.  The generic installation is described elsewhere.  Basically we are looking here at people being able to copy-paste a long apt-get series of commands to have REANA up and running.

(Along the same lines, please decide on usig either KVM2 and/or VirtualBox and then provide full copy-pastable instructions for only that hypervisor.  Again there is no need to be generic, we'll cover it elsewhere.  See e.g. how for macOS we mention only hyperkit.)

I'll revamp the Arch instructions anong the same line...

Reply:
I have changed accordingly and added all the commands that I needed to fully use REANA. The official guidelines contain more commands to check whether the installation ones are supported or not.